website/docs/dev/ide-setup.md

```diff
@@ -119,7 +119,7 @@ You can also format the whole project via Maven by using `mvn spotless:apply`.
 
 1. Go to "Settings" → "Tools" → "Checkstyle".
 2. Set "Scan Scope" to "Only Java sources (including tests)".
-3. For "Checkstyle Version" select "10.18.2".
+3. For "Checkstyle Version" select "9.6".
```

Comment:
What IDEA version are you using? I'm using `IntelliJ IDEA 2024.3.1.1 (Ultimate Edition)` and Checkstyle-IDEA 5.100.0, but can't find the "Checkstyle Version" for `9.6`, only `9.3` is available. Besides, Flink 1.20 also suggests `9.3`: https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/flinkdev/ide_setup/#checkstyle-for-java

Reply:
> What IDEA version are you using? I'm using `IntelliJ IDEA 2024.3.1.1 (Ultimate Edition)` and Checkstyle-IDEA 5.100.0, but can't find the "Checkstyle Version" for `9.6`, only `9.3` is available. Besides, Flink 1.20 also suggests `9.3`: https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/flinkdev/ide_setup/#checkstyle-for-java

Sorry for the wrong version. I truly using 9.3. I have updated the correct version. PTAL.